arduino/libraries/loader.go

```diff
@@ -184,7 +184,9 @@ func addExamplesToPathList(examplesPath *paths.Path, list *paths.PathList) error
 		_, err := sketch.New(file)
 		if err == nil {
 			list.Add(file)
-		} else if file.IsDir() {
+		} else if isDir, err := file.IsDirCheck(); err != nil {
+			return err
+		} else if isDir {
```

Comment:
I don't understand why this change fixes the issue, can you explain it a bit?
Is `file.IsDir()` returning `true` on symlinks? why `IsDirCheck` would return error instead? 🤔